ui/apps/platform/src/Containers/Policies/Wizard/Step3/policyCriteriaDescriptors.tsx

```diff
@@ -475,12 +475,33 @@ export const policyConfigurationDescriptor: Descriptor[] = [
         canBooleanLogic: true,
         lifecycleStages: ['BUILD', 'DEPLOY', 'RUNTIME'],
     },
+    {
+        label: 'CVE Is Fixable',
+        name: 'CVE Is Fixable',
+        shortName: 'CVE Is Fixable',
+        longName: 'CVE Is fixable',
+        category: policyCriteriaCategories.IMAGE_CONTENTS,
+        type: 'radioGroup',
+        radioButtons: [
+            {
+                text: 'CVE is fixable',
+                value: true,
+            },
+            {
+                text: 'CVE is not yet fixable',
```

Comment:
Is the meaning just as clear without **yet**? It adds a time dimension instead of plain true/false distinction.

Any extra modifier is possible source of confusion for a global audience.